cobalt/browser/global_features.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COBALT_BROWSER_GLOBAL_FEATURES_H_
+#define COBALT_BROWSER_GLOBAL_FEATURES_H_
+
+#include "base/memory/raw_ptr.h"
+#include "base/sequence_checker.h"
+#include "cobalt/browser/metrics/cobalt_metrics_services_manager_client.h"
+#include "components/metrics_services_manager/metrics_services_manager.h"
+
+namespace cobalt {
+
+class GlobalFeatures {
```

Comment:
Add a top level comment on what this class is and why it exists. Perhaps also mention it's relationship to browser_process.h on the Chrome side.